src/style.css

```diff
@@ -1 +1,25 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
```

Comment:
You have applied margin to the right of each star. This is not consistent with the guideline to apply margin only to the top or bottom of elements. Consider changing this to margin-bottom and adjusting your layout accordingly.